script/build-module.js

```diff
@@ -117,6 +124,34 @@ async function parseFile (file) {
   return cleanDeep(file)
 }
 
+function splitMd (md) {
+  const sections = []
+  let section = { name: null, body: [] }
+  let inCodeBlock = false
+  const isHeading = (line) => (!inCodeBlock && line.trim().startsWith('#'))
+
+  md.split('\n').forEach((curr, i, lines) => {
+    if (curr.startsWith('```')) {
+      inCodeBlock = !inCodeBlock
+    }
+    if (isHeading(curr)) {
```

Comment:
I would recommend using [slugg](https://ghub.io/slugg) for this. We're already using it in `electron-apps` and elsewhere.

Reply:
Cool! I didn't know about this!